src/main/kotlin/org/rust/openapiext/utils.kt

```diff
@@ -176,7 +176,7 @@ val VirtualFile.document: Document?
     get() = FileDocumentManager.getInstance().getDocument(this)
 
 val PsiFile.document: Document?
-    get() = PsiDocumentManager.getInstance(project).getDocument(this)
+    get() = viewProvider.document
```

Comment:
What does `PsiDocumentManager.getInstance(project).getDocument(this)` return in the case of non physical copy of `PsiFile`?

Reply:
It returns `null`. New approach is recommended by [Inspection preview guidelines](https://docs.google.com/document/d/1o-aRu8p_gemOl6O0CuWplsP3lJ2T4gS5zVQ9-cTrXz4/edit#heading=h.t6xf05a5xa09) (point `3e`)